examples/plotSkyMap.py

```diff
@@ -38,7 +40,7 @@
 
 def reportSkyMapInfo(skyMap):
     paramDict = skyMap.config.toDict()
-    paramNameList = sorted(paramDict.iterkeys())
+    paramNameList = sorted(paramDict.keys())
```

Comment:
I think the `keys` is not needed either. If you do `sorted(paramDict)` I think you get the keys sorted.